superflore/generators/bitbake/run.py

```diff
@@ -82,6 +88,24 @@ def main():
             os.chmod(_repo, 17407)
         # clone if args.output-repository_path is None
         overlay = RosMeta(_repo, not args.output_repository_path)
+        if not args.only:
+            pr_comment = pr_comment or (
+                'Superflore yocto generator began regeneration of all ' +
+                'packages form ROS distribution(s) %s from allenh1\'s ' +
```

Comment:
Can you make the upstream repo a parameter which defaults to `allenh` instead of having it hard coded?

Reply:
Absolutely. I wanted to add a flag that can redirect the PR in a follow up for both Gentoo and Open Embedded (a `--repository-url` flag).